### ISSUE TITLE
Lost all data, not able to add new accounts

### ISSUE THREAD

**Issue by kazigk** [raivo-otp/ios-application#332]
After recent update, I lost all my data. I can see 2 more updates were released in a rush trying to fix the issue.

The app is still in an unusable state and suddenly requires subscription that can't even be purchased - nothing happens when you click on "Continue".

I can see that this app was acquired by some shady company, isn't open-source anymore and requires subscription after the recent updates.

Thankfully I have a backup and I'm already switching to a different app.

**Comment by Tmp341** [comment]
If you have backed up your data via iCloud as ZIP file, you need to extract that first (need your master password). Then you can move your totps to [Ente Auth](https://ente.io/auth/). It has restore feature for Raivo OTP.

**Comment by Rchtr** [comment]
Why do you all prefer Ente Auth?
I do not get it, why youn have to make an account?

What about 2Fas? What is the difference between 2Fas and Ente Auth?

**Comment by zSydor** [comment]
> If you have backed up your data via iCloud as ZIP file, you need to extract that first (need your master password). Then you can move your TOTPs to [Ente Auth](https://ente.io/auth/). It has restore feature for Raivo OTP **json** files.

Now they want money you can't export

**Comment by helmarw** [comment]
yes its a pity its behind a paywall now, was bound to happen at some point i guess...
20$/year is also reasonable, there are more expensive auth apps out there (but also some free ones still)
but the payment system doesnt work (yet?!) and i cant add new 2FA.
Good that a read about the problems before and did a backup.
Now i trying a few different authenticators to see if i can and want to switch

**Comment by zSydor** [comment]
I am switching to 2FAS, luckily i didn't lose all my codes

**Comment by Tmp341** [comment]
> Now they want money you can't export

Ofc. I meant *previously*. They are forcing people to pay now. Not to mention constant app store rating request.

**Comment by Tmp341** [comment]
> I am switching to 2FAS, luckily i didn't lose all my codes

2FAS is iOS 16.4+, so i can't use it. :(

**Comment by zSydor** [comment]
> > Now they want money you can't export
> 
> Ofc. I meant _previously_. They are forcing people to pay now. Not to mention constant app store rating request.

Sorry i just woke up and didn't read properly

> > I am switching to 2FAS, luckily i didn't lose all my codes
> 
> 2FAS is iOS 16.4+, so i can't use it. 😞

Ente Auth it's ios 12+, opensource

**Comment by kazigk** [comment]
> If you have backed up your data via iCloud as ZIP file, you need to extract that first (need your master password). Then you can move your TOTPs to [Ente Auth](https://ente.io/auth/). It has restore feature for Raivo OTP **json** files.

I do, and I was even able to import that back to Raivo, but I can't add any new account without paying, so yeah..

I see lots of people are suggesting Ente Auth and 2FAS. I will give Ente Auth a try, looks neat and supports all my platforms.

**Comment by Rchtr** [comment]
at everyone:

I was able to get access to all of my TOTPs. The paywall can be canceled in the top left corner. After it has been canceled, you can restore your data from iCloud Backup (if you have saved it in an old version).

But it is not possible to export them without paying!
